plugin-gradle/src/test/java/com/diffplug/gradle/spotless/DiffMessageFormatterTest.java

```diff
@@ -151,10 +151,10 @@ void customRunToFixMessage() throws Exception {
 	@Test
 	void whitespaceProblem() throws Exception {
 		Bundle spotless = create(setFile("testFile").toContent("A \nB\t\nC  \n"));
-		spotless.task.addStep(FormatterStep.createNeverUpToDate("trimTrailing", input -> {
+		spotless.task.setSteps(List.of(FormatterStep.createNeverUpToDate("trimTrailing", input -> {
```

Comment:
I'm tempted to revert this code change and all other `addStep` -> `setStep` changes. WDYT?

Reply:
I can see an argument for `setSteps(FormatterStep... steps)` to remove the manual `List.of`, but I think the mutable-naming `addStep` is bad. It's used only for a few tests, which is why I just did the `List.of`.

Reply:
Fair enough! I have no strong opinions either way, so I'm happy to continue with `setSteps`.